predicators/envs/spot_env.py

```diff
@@ -726,6 +727,11 @@ def _blocking_classifier(state: State, objects: Sequence[Object]) -> bool:
     if _object_in_xy_classifier(state, blocker_obj, blocked_obj):
         return False
 
+    spot, = state.get_objects(_robot_type)
+    if blocked_obj.is_instance(_movable_object_type) and \
+        _holding_classifier(state, [spot, blocked_obj]):
+        return False
```

Comment:
Wait - why is this necessary? Does it have to do with the way that `DragToUnblock` works?

Reply:
Unfortunately I can't remember the case that spurred this change, but I think it was some unusual edge case where we classified an object that we were holding to be blocking another object. I don't think it was even related to sweeping or dragging, it just came up in the course of testing this PR...